client/src/components/FavoriteCoins.js

```diff
@@ -0,0 +1,141 @@
+import styled from 'styled-components/macro';
+import PropTypes from 'prop-types';
+import favoriteIcon from '../image/star.png';
+
+export default function FavoriteCoinsCard({
```

Comment:
here you could also consider to match the name of the component with the actual file name